### ISSUE TITLE
Parameter type List<AWS::EC2::SecurityGroup::Id>

### ISSUE THREAD

**Issue by martysweet** [martysweet/cfn-lint#94]
```yaml
AWSTemplateFormatVersion: '2010-09-09'
Parameters:
  ExistingSecurityGroups:
    Type: List<AWS::EC2::SecurityGroup::Id>
Resources:
  Bucket:
    Type: AWS::S3::Bucket
```

Results in:
```
0 infos
0 warn
1 crit
Resource: Parameters > ExistingSecurityGroups
Message: Parameter ExistingSecurityGroups has an invalid type of List<AWS::EC2::SecurityGroup::Id>.
Documentation: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html
```

However `List<AWS::EC2::SecurityGroup::Id>` is valid.

cfn-lint@1.3.4
